src/test/java/build/buildfarm/common/DigestUtilTest.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
copyright date

Reply:
I must not copy/paste
I must not copy/paste
I must not copy/paste
I must not copy/paste